src/backend/routers/case_manager.ts

```diff
@@ -77,6 +77,46 @@ export const case_manager = router({
         .executeTakeFirstOrThrow();
     }),
 
+  // TODO: create editStudent authenticated procedure. This is a possible example, but NOT WORKING YET. Also, could be .updateTable instead of .update
```

Comment:
is this procedure working?

Reply:
Sorry, my bad! Forgot to update that comment in the original commit for this PR.

Reply:
Yes, the procedure is working fine.